test/k8sT/Tunnels.go

```diff
@@ -103,16 +110,14 @@ var _ = Describe("K8sValidatedTunnelTest", func() {
 			By("Checking that BPF tunnels are working correctly")
 			tunnStatus := isNodeNetworkingWorking(kubectl, "zgroup=testDS")
 			Expect(tunnStatus).Should(BeTrue())
+
+			// FIXME need to be sync when Cilium restores
```

Comment:
I filed GH-4457 so this can be referenced via a GH issue instead of just a FIXME. Please update the comment to reference this GH issue.

Reply:
done